jupyterlab_caip_optimizer/src/components/study_details.tsx

```diff
@@ -0,0 +1,292 @@
+import React from 'react';
+import {
+  Box,
+  Button,
+  Paper,
+  Typography,
+  Grid,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from '@material-ui/core';
+import { withStyles, makeStyles } from '@material-ui/core/styles';
+import * as Types from '../types';
+import { prettifyStudyName } from '../service/optimizer';
+import { connect } from 'react-redux';
+import moment from 'moment';
+import { dateFormat, makeReadable } from '../utils';
+import { setView } from '../store/view';
+
+const StyledTableCell = withStyles(theme => ({
+  head: {
+    backgroundColor: theme.palette.action.hover,
+    color: theme.palette.common.black,
+  },
+  body: {
+    fontSize: 14,
+  },
+}))(TableCell);
+
+const StyledTableRow = withStyles(theme => ({
+  root: {
+    '&:nth-of-type(odd)': {
+      backgroundColor: theme.palette.common.white,
+    },
+  },
+}))(TableRow);
+
+const useStyles = makeStyles({
+  table: {
+    minWidth: 700,
+  },
+});
+
+interface FormattedParam {
+  name: string;
+  type: Types.ParameterType;
+  values?: number[] | string[];
+  minValue?: number | string;
+  maxValue?: number | string;
+}
+
+interface FormattedStudy {
+  name: Types.Study['name'];
+  objective: string;
+  algorithm: Types.StudyConfig['algorithm'];
+  state: Types.Study['state'];
+  createTime: Types.Study['createTime'];
+  parameters: FormattedParam[];
+}
+
+function formatParams(params: Types.ParameterSpec[]): FormattedParam[] {
+  const formattedParams: FormattedParam[] = params.map(param => {
```

Comment:
nit; You can probably just put the return here instead of declaring a const beforehand

Reply:
Fixed it :)